packages/docusaurus-theme-common/src/utils/usePluralForm.ts

```diff
@@ -65,7 +65,8 @@ function useLocalePluralForms(): LocalePluralForms {
     i18n: {currentLocale},
   } = useDocusaurusContext();
   return useMemo(() => {
-    if (Intl && Intl.PluralRules) {
+    // @ts-expect-error checking Intl.PluralRules in case browser doesn't have it (e.g Safari 12-)
+    if (Intl.PluralRules) {
```

Comment:
so it is safe to assume that Intl is also present in older browsers right?

Reply:
yeah since IE11